kalite/distributed/features/steps/superuser_create.py

```diff
@@ -0,0 +1,124 @@
+from behave import *
+from selenium import webdriver
+from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
+
+modal_container = "superusercreate-container"
+
+@given("there is superuser")
+def step_impl(context):
+    if User.objects.exists():
+        pass
+    else:
+        User.objects.create_superuser(username='superusername', password='superpassword', email='super@email.com')
+    assert User.objects.exists(), "superuser not crerated successfully!"
+
+@then("there should be no modal displayed")
+def step_impl(context):
+    assert not find_id_with_wait(context, modal_container), "modal found!"
+
+@given("there is no superuser")
+def step_impl(context):
+    if User.objects.exists():
+        User.objects.all().delete()
+    assert not User.objects.exists(), "superuser not deleted successfully!"
+
+@then("I should see a modal")
+def step_impl(context):
+    assert find_id_with_wait(context, modal_container).is_displayed(), "modal not displayed!"
+
+@then("I will not be able to dismiss the modal until a superuser is created")
+def step_impl(context):
+    # don't know how to test this
+    pass
+
+@given("the username is empty")
+def step_impl(context):
+    fill_username(context, "")
+
+@when('I click the create button')
+def step_impl(context):
+    create_button = find_css_class_with_wait(context, "create-btn")
+    create_button.click()
+    context.browser.implicitly_wait(1)
```

Comment:
This doesn't actually wait for anything, it just sets an implicit wait time for future calls to find elements.